# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/UsageInfo.h
# Commit: 868577854548
# Full Hash: 868577854548c7442c15ab9b1ad1213af187e01e
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-11-17 21:55:29
# Regressor Bug: 1671369
# File Overlap Count: 1
# Description:
#   Bug 1671369 - Change GetUsageForClient to return a UsageInfo. r=dom-workers-and-storage-reviewers,ttung
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93777
# ==============================================================================

diff -r 5f347d3846aa -r 868577854548 dom/quota/UsageInfo.h
--- a/dom/quota/UsageInfo.h	Tue Nov 17 12:13:05 2020 +0000
+++ b/dom/quota/UsageInfo.h	Tue Nov 17 13:17:24 2020 +0000
@@ -84,6 +84,8 @@
     return *this;
   }
 
+  Maybe<uint64_t> DatabaseUsage() const { return mDatabaseUsage.GetValue(); }
+
   Maybe<uint64_t> FileUsage() const { return mFileUsage.GetValue(); }
 
   Maybe<uint64_t> TotalUsage() const {
